Azordev/good-github-issues
# Expanding doc for freelancers

## WIP.

I'm still working in this issue, but if you have time you can help me checking how I go, and let me comments about how to improve.

## Explanation

I've added a best documentation for freelancers that want to come to work with Azordev. That I want is that everyone that read this can start to work with us without need any meet.

## TO DO:

- [x] Doc for PM's
- [ ] Doc for QA's
    - [x] Creating issues
    - [ ] Testing websites in search of bugs
- [x] Doc for Developers Freelancers
    - [ ] Doc about how to work using LiveShare
    - [ ] Doc about how to connect to another PC using Chrome Remote Desktop
- [ ] Doc about payments
- [ ] General README
- [ ] Add english translates

## How to test

1. Go to: https://github.com/Azordev/good-github-issues/tree/feat/expanding-doc-2
2. Enter at the folders and let me know if is entendible the way that I have write the documentation

Everyone can help me to check if the doc is good or if need some more, I'm open to listen ideas to improve the documentation.
